.eslintrc

```diff
@@ -18,7 +18,9 @@
       "files": [ "**/*.ts" ],
       "extends": [
         "eslint:recommended",
-        "prettier"
+        "prettier",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"    
```

Comment:
What is the difference between `eslint-recommended` and `recommended`?

Reply:
Please Tim we cannot keep on opening topics. It is the recommended configuration I have seen in many web pages. You can go into the node_modules and see what they configure. We have already decided to proceed with them and disable topics we do not agree with.